checker/tests/calledmethods/EnsuresCalledMethodsOnExceptionRepeatable.java

```diff
@@ -0,0 +1,43 @@
+import java.io.*;
```

Comment:
@kelloggm is there a convention / rule around adding a comment at the top of test input source files indicating what is being tested?

Reply:
I generally try to do so.

If the test case corresponds to a bug that was reported in the issue tracker, the convention is for the first line to be something like:
```
// test case for [URL of GitHub issue]
```

Reply:
This seems like a good practice. I've added some descriptions for each new test.